### DISCUSSION TITLE
Work with Strimzi in namespaces where exists another resources

### DISCUSSION THREAD

**Discussion by SamMc75** [strimzi/strimzi-kafka-operator#9845] [Q&A]
Hi! We need use Strimzi in namespaces where exists another resources (for example pods). Important that strimzi-cluster-operator will not can manage (get, list, update, delete, watch) outsider resources. Can you help us to know how we have to tune ClusterRole and RoleBinding for this?

**Comment by scholzj** [comment]
Strimzi will not _manage_ any other resources unless they have some conflicting names or labels.

I'm not sure you can do this easily with RBAC. You might try to figure out all the desired names and set the RBAC rights only for those. But TBH you would have to figure it out your self -. probably by trying it and fixing the errors as they come.
